### PR TITLE
Pass image height and width to client

### PR DESCRIPTION
**Pull request by webb04**
Fix for GLA-2121 where some images have an incorrect ratio in the slideshow/lightbox view.

### CODE REVIEW

File: src/client/nativeCommunication.ts [hunk 1]
```diff
@@ -108,7 +111,9 @@ function launchSlideshow(src: string | null): void {
             const url = updateUrl(image?.currentSrc ?? image.src);
             const caption =  image.getAttribute('data-caption') ?? undefined;
             const credit = image.getAttribute('data-credit') ?? undefined;
-            return new Image({ url, caption, credit });
+            const width = getImageWidth(url);
+            const height = width * parseFloat(image.getAttribute('data-ratio') ?? '0.56');
```
Comment (JamieB-gu) [inline src/client/nativeCommunication.ts:115]: It's possible height could be `NaN` here. What happens in the Thrift transport layer/on the native side if a number value is `NaN`?

Comment (webb04) [reply src/client/nativeCommunication.ts:115]: Could it? `getImageWidth` returns a number and `image.width` `image.height` aren't optional

Comment (JamieB-gu) [reply src/client/nativeCommunication.ts:115]: `parseFloat` can return a `NaN`, which could happen, for example, if `data-ratio` gets mangled.

Comment (webb04) [reply src/client/nativeCommunication.ts:115]: ok we might have to avoid showing the image without having a height/width.

**Pull request merged**